### DISCUSSION TITLE
Would anyone be able to add 'location created' to the video class?

### DISCUSSION THREAD

**Discussion by nikolkhb** [davidteather/TikTok-Api#947] [Ideas]
Hi everyone,

I'm able to pull the location the video was created in from TikTok's source code (see image attached - 'locationcreated'). It's right next to 'adAuthorization' and 'adLabel' attributes. However, when pulling in 'stats' or 'info' from the TikTokApi from the 'video' class, the data returned only shows the adAuthorization and adLabel, but not locationcreated. Please can the code be updated to reflect location created as I am trying to model TikTok audience demographics, or can someone kindly suggest a workaround? :) 

![location_created_ss](https://user-images.githubusercontent.com/113099312/189155574-5c260562-89a1-42be-a255-ce2fad0905d7.png)

Thanks in advance!

**Comment by davidteather** [comment]
You could make HTTP calls directly and extract the data from the HTML for the videos.
If you're using V6 you could do something like
```py
await api.sessions[0].page.goto(the_video_url)
data = api.sessions[0].page.content # should pull the html on screen
```
